flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/services/TablePartitionInfo.java

```diff
@@ -18,24 +18,34 @@
 
 import org.apache.flink.annotation.Internal;
 
+import org.apache.flink.shaded.guava30.com.google.common.collect.Lists;
```

Comment:
We can avoid this guava usage. Are we addressing this later?

Reply:
Yea PR #60

Reply:
Yes.